modules/apps/layout/layout-page-template-item-selector-web/src/main/java/com/liferay/layout/page/template/item/selector/web/internal/LayoutPageTemplateCollectionsItemSelectorView.java

```diff
@@ -109,11 +108,6 @@ public void renderHTML(
 	private LayoutPageTemplateCollectionLocalService
 		_layoutPageTemplateCollectionLocalService;
 
-	@Reference(
-		target = "(osgi.web.symbolicname=com.liferay.layout.page.template.item.selector.web)"
-	)
-	private ServletContext _servletContext;
```

Comment:
@JanisZhang Let's keep `ServletContext` for now. If you see any unused `ServletContext` reference, please record it.

`ServletContext` is not just a class that provides some APIs, it also represents a state of the web module.

Also, most of other `ItemSelectorView` implementations also have `ServletContext`, and those who extends `BaseLayoutsItemSelectorView` has methods that provides the referenced `ServletContext`.